05LessonFive/Checkpoint1/index.html

```diff
@@ -9,14 +9,18 @@
 <body>
     <table>
         <tr>
+            <!--picture of berries-->
             <td width="500px">
                 <img src="img/berries.jpg" width="500px" />
             </td>
+            <!--box with text and link-->
             <td class="grey_box" width="500px">
                 <h1 align="center">Berries</h1>
                 <p align="center">
                     The snozzberries taste like snozzberries! You should totally read this article on Cracked.com! Childhood ruined. You're welcome.
                     <center>
+
+                        <!--read more button-->
```

Comment:
I think I changed all the stuff you mentioned. I went with tables because I
didn't know how in the hell to do the float thing, but now that I'm looking
at some of my classmates' code, I can see that I've made this shit wayyyy
too complicated.

On Mon, May 16, 2016 at 4:25 PM, Jake Bapple <notifications@github.com>
wrote:

> In 05LessonFive/Checkpoint1/index.html
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/33#discussion_r63428659>
> :
>
> >              <td class="grey_box" width="500px">
> >                  <h1 align="center">Berries</h1>
> >                  <p align="center">
> >                      The snozzberries taste like snozzberries! You should totally read this article on Cracked.com! Childhood ruined. You're welcome.
> >                      <center>
> > +
> > +                        <!--read more button-->
>
> Great job commenting! any reason you went with tables? also, thanks for
> ENHANCING my childhood. the fact that snozzberries are dicks is pretty
> great.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/33/files/219d75a5488135e003574ccc79c2441581d7db3c..3c054a43b0132d367990ab749a304c941716953d#r63428659>
>